user.css

```
@@ -685,4 +685,107 @@ body.spotui-search-mode #spotui-back-btn {
+    display: flex;
+    align-items: center;
+    gap: 12px;
+    min-width: 0;
+}
+
+.spotui-custom-bar-heart {
+    cursor: pointer;
+    appearance: none;
+    -webkit-appearance: none;
+    border: none;
+    background: none;
```

Stylelint (17.14.0)

[error] Duplicate property "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)